pkg/arvo/app/link-listen-hook.hoon

```diff
@@ -62,11 +62,20 @@
     ~|  [dap.bowl %weird-wire wire]
     !!
   ::
+  ++  on-arvo
+    |=  [=wire =sign-arvo]
+    ^-  (quip card _this)
+    ?.  ?=([%g %done *] sign-arvo)
+      (on-arvo:def wire sign-arvo)
+    ?~  error.sign-arvo  [~ this]
+    =/  =tank  leaf+"{(trip dap.bowl)}'s message went wrong!"
+    %-  (slog tank tang.u.error.sign-arvo)
+    [~ this]
```

Comment:
@philipcmonk I was kind of surprised to need to implement this (to catch `[%g %done ...]`), would've expected network messages to apps ack via `+on-agent` instead...